test/Microsoft.TestPlatform.AcceptanceTests/BlameDataCollectorTests.cs

```diff
@@ -23,6 +24,19 @@ public class BlameDataCollectorTests : AcceptanceTestBase
 {
     public const string NETCOREANDFX = "net452;net472;netcoreapp3.1";
     public const string NET50 = "net5.0";
+    private readonly string _procDumpPath;
+
+    public BlameDataCollectorTests()
+    {
+        _procDumpPath = Path.Combine(_testEnvironment.PackageDirectory, @"procdump\0.0.1\bin");
+        var procDumpExePath = Path.Combine(_procDumpPath, "procdump.exe");
+        if (!File.Exists(procDumpExePath))
+        {
+            throw new InvalidOperationException($"Procdump path {procDumpExePath} does not exist. "
+                + $"It is possible that antivirus deleted it from your nuget cache. "
+                + $"Delete the whole procdump folder in your nuget cache, and run build, or restore");
```

Comment:
[minor] Interpolation strings are not needed.

```suggestion
                + "It is possible that antivirus deleted it from your nuget cache. "
                + "Delete the whole procdump folder in your nuget cache, and run build, or restore");
```